### ISSUE TITLE
BangFormat false positive

### ISSUE THREAD

**Issue by steffenweber** [brigade/scss-lint#289]
```
$ cat test.css 
.foo {
    content: '!';
}

$ scss-lint test.css 
test.css:2 [W] BangFormat: ! should be preceeded by a space, and should not be followed by a space
```

In this case it's fine to not preceed the exclamation mark by a space. :smiley:
